### ISSUE TITLE
proto files not used

### ISSUE THREAD

**Issue by sinag** [BlueMatthew/ProtobufInspector#2]
Hi,

Even if I have matching .proto files, message types are not visible in messagetype combobox.
I can see decompiled proto in treeview, but not with messagetypes
Any hints?

**Comment by BlueMatthew** [comment]
Yes, the message type combo-box is only for manual input when the program can't get message type from http response. I wonder if your data returns message type? If it does, the message type will be filled into the combo-box.

**Comment by BlueMatthew** [comment]
> Hi,
> 
> Even if I have matching .proto files, message types are not visible in messagetype combobox. I can see decompiled proto in treeview, but not with messagetypes Any hints?

Please check the decompiled proto in treeview that field name is string defined in proto or pure number(field id)? if it is number, the data is decoded as raw message.
